```
[01:47:27.691925 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38349)	=================================== FAILURES ===================================	
[01:47:27.691940 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38350)	___________________ test_jitify_deprecation_warning[True-.*] ___________________	
[01:47:27.691942 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38351)		
[01:47:27.691946 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38352)	jitify = True, match = '.*'	
[01:47:27.691956 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38353)		
[01:47:27.691958 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38354)	    @pytest.mark.parametrize("jitify,match", [	
[01:47:27.691964 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38355)	        (True, ".*"),	
[01:47:27.691987 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38356)	        (False, "Avoid passing.*jitify=False")	
[01:47:27.691990 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38357)	    ])	
[01:47:27.691994 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38358)	    @pytest.mark.thread_unsafe(reason="uses temporary cache dir")	
[01:47:27.692006 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38359)	    @use_temporary_cache_dir()	
[01:47:27.692009 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38360)	    def test_jitify_deprecation_warning(jitify, match):	
[01:47:27.692022 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38361)	        with pytest.warns(DeprecationWarning, match=match):	
[01:47:27.692035 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38362)	            cupy.RawKernel(	
[01:47:27.692039 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38363)	                _test_source1, 'test_sum',	
[01:47:27.692047 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38364)	                backend='nvrtc', jitify=jitify)	
[01:47:27.692057 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38365)		
[01:47:27.692059 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38366)	        with pytest.warns(DeprecationWarning, match=match):	
[01:47:27.692065 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38367)	            cupy.RawModule(	
[01:47:27.692085 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38368)	                code=_test_source1,	
[01:47:27.692088 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38369)	                backend='nvrtc', jitify=jitify)	
[01:47:27.692092 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38370)		
[01:47:27.692102 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38371)	        # Not technically part of the rawkernel, but test warning in compile here:	
[01:47:27.692105 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38372)	        with pytest.warns(DeprecationWarning, match=match):	
[01:47:27.692109 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38373)	>           compiler.compile_using_nvrtc("", options=(), jitify=jitify)	
[01:47:27.692120 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38374)		
[01:47:27.692124 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38375)	jitify     = True	
[01:47:27.692131 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38376)	match      = '.*'	
[01:47:27.692144 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38377)		
[01:47:27.692151 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38378)	cupy_tests/core_tests/test_raw.py:1468:	
[01:47:27.692157 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38379)	_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _	
[01:47:27.692171 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38380)	/root/.local/lib/python3.12/site-packages/cupy/cuda/compiler.py:403: in compile_using_nvrtc	
[01:47:27.692173 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38381)	    return _compile_using_nvrtc_no_warning(	
[01:47:27.692178 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38382)	        arch       = None	
[01:47:27.692215 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38383)	        cache_in_memory = False	
[01:47:27.692219 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38384)	        filename   = 'kern.cu'	
[01:47:27.692224 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38385)	        jitify     = True	
[01:47:27.692237 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38386)	        log_stream = None	
[01:47:27.692241 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38387)	        method     = None	
[01:47:27.692247 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38388)	        name_expressions = None	
[01:47:27.692260 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38389)	        options    = ()	
[01:47:27.692263 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38390)	        source     = ''	
[01:47:27.692268 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38391)	/root/.local/lib/python3.12/site-packages/cupy/cuda/compiler.py:391: in _compile_using_nvrtc_no_warning	
[01:47:27.692281 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38392)	    return _compile(source, options, cu_path, name_expressions,	
[01:47:27.692283 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38393)	        _compile   = <function _compile_using_nvrtc_no_warning.<locals>._compile at 0x7f5bc21c0680>	
[01:47:27.692288 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38394)	        arch       = None	
[01:47:27.692299 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38395)	        cache_in_memory = False	
[01:47:27.692303 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38396)	        cu_file    = <_io.TextIOWrapper name='/tmp/tmper3mrcu4/kern.cu' mode='w' encoding='utf-8'>	
[01:47:27.692312 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38397)	        cu_path    = '/tmp/tmper3mrcu4/kern.cu'	
[01:47:27.692319 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38398)	        filename   = 'kern.cu'	
[01:47:27.692323 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38399)	        jitify     = True	
[01:47:27.692332 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38400)	        log_stream = None	
[01:47:27.692340 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38401)	        method     = None	
[01:47:27.692343 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38402)	        name_expressions = None	
[01:47:27.692351 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38403)	        options    = ()	
[01:47:27.692356 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38404)	        root_dir   = '/tmp/tmper3mrcu4'	
[01:47:27.692363 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38405)	        source     = ''	
[01:47:27.692383 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38406)	/root/.local/lib/python3.12/site-packages/cupy/cuda/compiler.py:360: in _compile	
[01:47:27.692386 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38407)	    options, headers, include_names = _jitify_prep(	
[01:47:27.692392 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38408)	        arch       = None	
[01:47:27.692410 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38409)	        arch_opt   = '-arch=sm_75'	
[01:47:27.692420 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38410)	        cu_path    = '/tmp/tmper3mrcu4/kern.cu'	
[01:47:27.692430 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38411)	        jitify     = True	
[01:47:27.692433 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38412)	        log_stream = None	
[01:47:27.692437 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38413)	        method     = 'cubin'	
[01:47:27.692450 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38414)	        name_expressions = None	
[01:47:27.692454 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38415)	        options    = ('-arch=sm_75',)	
[01:47:27.692458 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38416)	        source     = ''	
[01:47:27.692467 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38417)	/root/.local/lib/python3.12/site-packages/cupy/cuda/compiler.py:285: in _jitify_prep	
[01:47:27.692470 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38418)	    jitify._init_module()	
[01:47:27.692478 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38419)	        core       = <module 'cupy._core.core' from '/root/.local/lib/python3.12/site-packages/cupy/_core/core.cpython-312-x86_64-linux-gnu.so'>	
[01:47:27.692506 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38421)	        cu_path    = '/tmp/tmper3mrcu4/kern.cu'	
[01:47:27.692511 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38422)	        jitify     = <module 'cupy.cuda.jitify' from '/root/.local/lib/python3.12/site-packages/cupy/cuda/jitify.cpython-312-x86_64-linux-gnu.so'>	
[01:47:27.692540 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38424)	        options    = ('-arch=sm_75',)	
[01:47:27.692543 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38425)	        source     = ''	
[01:47:27.692552 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38426)	cupy/cuda/jitify.pyx:220: in cupy.cuda.jitify._init_module	
[01:47:27.692554 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38427)	    ???	
[01:47:27.692557 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38428)	cupy/cuda/jitify.pyx:244: in cupy.cuda.jitify._init_module	
[01:47:27.692563 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38429)	    ???	
[01:47:27.692566 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38430)	cupy/cuda/jitify.pyx:217: in cupy.cuda.jitify._init_cupy_headers	
[01:47:27.692572 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38431)	    ???	
[01:47:27.692591 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38432)	cupy/cuda/jitify.pyx:206: in cupy.cuda.jitify._init_cupy_headers_from_scratch	
[01:47:27.692596 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38433)	    ???	
[01:47:27.692603 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38434)	cupy/cuda/jitify.pyx:112: in cupy.cuda.jitify.dump_cache	
[01:47:27.692638 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38435)	    ???	
[01:47:27.692644 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38436)	_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _	
[01:47:27.692650 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38437)		
[01:47:27.692671 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38438)	name = '/tmp/cupy_cache/.cupy/kernel_cache', mode = 511, exist_ok = True	
[01:47:27.692673 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38439)		
[01:47:27.692678 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38440)	>   ???	
[01:47:27.692705 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38441)	E   Failed: Timeout (>300.0s) from pytest-timeout.	
[01:47:27.692708 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38442)		
[01:47:27.692714 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38443)	exist_ok   = True	
[01:47:27.692733 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38444)	mode       = 511	
[01:47:27.692737 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38445)	name       = '/tmp/cupy_cache/.cupy/kernel_cache'	
[01:47:27.692745 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38446)		
[01:47:27.692767 STDOUT 1437]](https://ci.preferred.jp/cupy.linux.cuda129/208874/#L38447)	<frozen os>:200: Failed
```

--- a/tests/cupy_tests/core_tests/test_raw.py
+++ b/tests/cupy_tests/core_tests/test_raw.py
@@ -1450,6 +1450,9 @@ class TestRawJitifyJitify(_TestRawJitify, unittest.TestCase):
     (True, ".*"),
     (False, "Avoid passing.*jitify=False")
 ])
+@unittest.skipIf(cupy.cuda.runtime.is_hip,
+                 'Jitify does not support ROCm/HIP')
+@testing.slow
 @pytest.mark.thread_unsafe(reason="uses temporary cache dir")
 @use_temporary_cache_dir()
 def test_jitify_deprecation_warning(jitify, match):

